docs/docs-content/devx/apps/deploy-app.md

```diff
@@ -392,15 +392,50 @@ docker version
 Download the tutorial image to your local machine. <br />
 
 ```bash
-docker pull ghcr.io/spectrocloud/tutorials:1.1.2
+docker pull ghcr.io/spectrocloud/tutorials:1.1.3
 ```
 
 Next, start the container, and open a bash session into it.
 
 <br />
 
 ```shell
-docker run --name tutorialContainer --interactive --tty ghcr.io/spectrocloud/tutorials:1.1.2 bash
+docker run --name tutorialContainer --interactive --tty ghcr.io/spectrocloud/tutorials:1.1.3 bash
+```
+
+Navigate to the tutorial code.
+
+<br />
+
+```shell
+cd terraform/hello-universe-tf/
+```
+
+</TabItem>
+
+<TabItem label="Podman" value="Podman">
+
+Ensure Podman on your local machine is available. Use the following command and ensure you receive an output displaying
```

Comment:
@caroldelwing should we link to the podman machine start docs page? To help the reader out?
https://docs.podman.io/en/latest/markdown/podman-machine.1.html